synapse/tests/test_cortex.py

```diff
@@ -1042,6 +1042,13 @@ def onNodeDel(node):
             self.len(1, nodes)
             self.eq(nodes[0].ndef, ('inet:ipv4', 0x01020304))
 
+    async def test_cortex_axon(self):
+        async with self.getTestCore() as core:
+            await core.axready.wait()
+            size, sha2 = await core.axon.put(b'asdfasdf')
```

Comment:
have to add tests to test_axon.py for `put` and `puts` since those are currently untested functions.